### PR TITLE
PXP-659 feat: add cloud sql instances and metric clients from google protobuf…

### PR DESCRIPTION
**Pull request by Fadhil**
… specs

## Summary
In this PR, we generate the `google/cloud/sql/cloud_sql_instances` and `google/monitoring/metrics` proto clients needed for getting Cloud SLQ metrics (to be displayed in our CLI)

Ticket:

## What's Changed

Updated `build.rs` to include cloud_sql and metrics proto files to be compiled

Included generated clients for the above mentioned proto files.

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/191#" title="Author self-approved">Fadhil</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/191#" title="Author self-approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/191#pullrequestreview-1846788015" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
